os/centralized-logging.md

```diff
@@ -0,0 +1,138 @@
+# Centralized Logging
+
+Systmed Journald allows to transmit logs to central storage. Basic elements are:
```

Comment:
`Systemd journald` can be configured to store logs in a central repository. It can act both as a client, forwarding logs to a storage host, and as a server, collecting logs from remote clients and writing them to storage.